RealmSwift/Tests/RealmTests.swift

```diff
@@ -489,11 +493,13 @@ class RealmTests: TestCase {
         XCTAssertFalse(realm == testRealm)
 
         let fired = expectationWithDescription("fired")
-        dispatch_async(dispatch_queue_create("background", nil)) {
+        let queue = dispatch_queue_create("background", nil)
+        dispatch_async(queue) {
             let otherThreadRealm = Realm()
             XCTAssertFalse(realm == otherThreadRealm)
             fired.fulfill()
         }
         waitForExpectationsWithTimeout(2, handler: nil)
```

Comment:
The expectation here is pointless with the addition of the `dispatch_sync()` (in the other places it's needed to give the notification block a chance to run).